### ISSUE TITLE
Error RDFXX on a HTTP/1.1 307 TEMPORARY REDIRECT response

### ISSUE THREAD

**Issue by serasset** [openlink/virtuoso-opensource#1044]
I'm loading an ontology in a virtuoso instance that resolves using content negociation with HTTP1.1 307 Temporary Redirect response. 

## Behaviour in latest virtuoso release (from docker)

```
docker run -p9999:8890 -p2222:1111 openlink/virtuoso-opensource-7 start
```
then connect to the running container, start isql and evaluate:

```
$ $ISQL 1111 dba password
OpenLink Virtuoso Interactive SQL (Virtuoso)
Version 07.20.3233 as of Jun 22 2021
Type HELP; for help and EXIT; to exit.
Connected to OpenLink Virtuoso
Driver: 07.20.3233 OpenLink Virtuoso ODBC Driver
SQL> SPARQL LOAD <http://purl.org/olia/olia.owl> into graph <http://kaiko.getalp.org/datamodel>;

Done. -- 2047 msec.
SQL>
```

So far so good !

## Behaviour in develop/7 virtuoso release (using pkleef docker)

```
docker run -i -t pkleef/virtuoso-opensource-7:latest start
```
then connect to the running container, start isql and evaluate:

```
$ $ISQL 1111 dba password
OpenLink Virtuoso Interactive SQL (Virtuoso)
Version 07.20.3233 as of May 11 2022
Type HELP; for help and EXIT; to exit.
Connected to OpenLink Virtuoso
Driver: 07.20.3233 OpenLink Virtuoso ODBC Driver
SQL> SPARQL LOAD <http://purl.org/olia/olia.owl> into graph <http://kaiko.getalp.org/datamodel>;

*** Error RDFXX: [Virtuoso Driver][Virtuoso Server]Unable to retrieve data from "http://purl.org/olia/olia.owl": HTTP/1.1 307 TEMPORARY REDIRECT
at line 1 of Top-Level:
SPARQL LOAD <http://purl.org/olia/olia.owl> into graph <http://kaiko.getalp.org/datamodel>
SQL>
```

Expected behaviour: no error and proper file resolution

It also happens on a vanilla git pulled and locally compiled virtuoso instance.

**Comment by pkleef** [comment]
Ok, i will have a look.

**Comment by pkleef** [comment]
This is fixed in commit a01fb790c4f2953dcd92ad4d7782a17291bba345 and will be part of the imminent stable release.
